src/test/java/fi/metatavu/mecm/reader/vcard/ConverterTest.java

```diff
@@ -166,20 +166,24 @@ public void testStatus() throws IOException, JsonParseException, JsonMappingExce
   public void testPrivateCard() {
     VCard vCard1 = loadTestCard(0);
     VCard vCard2 = loadTestCard(1);
+    VCard vCard3 = loadTestCard(2);
 
     assertEquals("false", vCard1.getExtendedProperty(VCardConverter.MECM_ADDITIONAL_PRIVATE).getValue());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "false" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)